Makefile

```diff
@@ -71,10 +71,14 @@ METRICS_PORT ?= 9118
 export CAPELLA_VERSIONS ?= 5.0.0 5.2.0 6.0.0
 
 # Capella version used to run containers
-CAPELLA_VERSION ?= 6.0.0
+export CAPELLA_VERSION ?= 6.0.0
 
-# Only used when "capella_loop.sh" is NOT used
-export DOCKER_TAG=$(CAPELLA_VERSION)-$(CAPELLA_DOCKERIMAGES_REVISION)
+# Comma-separated list of dropins to download & add, doesn't affect copied & mounted dropins
+# See available options in documentation.
```

Comment:
Maybe add the link to the documentation page here

Reply:
Added.